hw2/04-background.js

```diff
@@ -1 +1,55 @@
 // Add your code here
+let intervalId;
+let backgroundColorInterval = 3000;
+
+const button = document.getElementById("startstop");
+const input = document.getElementById('input[type="text"]');
```

Comment:
This may be why you have issues; you're looking for an element with input type being text. However, the input for seconds is of type number ;)